src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1577,6 +1577,7 @@ IF(PLUS_USE_WINPROBE_VIDEO)
   ENDIF()
 
   FILE(GLOB WinProbeAssemblies "${WINPROBESDK_DIR}/${Bitness}/*.dll")
+  FILE(COPY ${WinProbeAssemblies} DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
I believe this is the purpose of `External_Libraries_Install` list. Could you add it to that so that other features that use this list will work?